bundles/core/src/test/java/com/adobe/cq/wcm/core/components/internal/servlets/DMAssetPostProcessorTest.java

```diff
@@ -362,6 +362,45 @@ private void assertNewModification(String component, List<Modification> modifica
         }
     }
 
+    /*
+     The case when DM asset drug-n-drop on empty page. This scenario creates Image component with assigned asset.
+     Post processor should write image server url property
+     */
+    @ParameterizedTest
+    @EnumSource(
+        value = ModificationType.class,
+        names = {"CREATE", "MODIFY"}
+    )
+    public void drugNdropDMimageToEmptyPage(ModificationType modificationType) throws Exception {
+        String existingComponent = "/content/test/jcr:content/root/container";
+        prepareResource(existingComponent, null);
+        List<Modification> modifications = prepareModifications(modificationType, "/content/test/jcr:content/root/container/image40", true);
+        servlet.process(context.request(), modifications);
+        Resource resource = context.currentResource("/content/test/jcr:content/root/container/image40");
+        assertNotNull(resource);
+        assertEquals(EXPECTED_IMAGE_SERVER_URL, resource.getValueMap().get(Image.PN_IMAGE_SERVER_URL, String.class));
```

Comment:
@oborysyu add check which makes sure that Image.PN_IMAGE_SERVER_URL is NOT created on responsiveGrid resource